book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ OPTIONAL] For better code organization, it will be nice to move these variables to the top of the class.